.github/workflows/issue-triage.yml

```diff
@@ -0,0 +1,28 @@
+# label management triage | good first issue
+#                      -> untriaged|duplicate|invalid
+#                         ->bug|enhancement|feature-request|documentation|question|wontfix|Known issue
+#                           -> up-for-grabs | in progress | with repo | needs repo | tracking-external-issue
+# https://github.com/coverlet-coverage/coverlet/blob/master/Documentation/Roadmap.md
```

Comment:
I think those comments don't really describe our current workflow. Usually we add the label `untriaged` to new issues and then adjust the labels if somebody found time to analyze it. If you wan't to change the current workflow then we should discuss it and document it consistently. E.g. the workflow in the comment doesn't really match the description in the roadmap document.

Reply:
I will keep the list of labels and the (Roadmap.md )(https://github.com/coverlet-coverage/coverlet/blob/master/Documentation/Roadmap.md) URL . This document has some details for labels but not for all of them.

By the way, I can see the list of labels only since I am a co-maintainer.